src/lib/bootloader/bootloader_factory.rb

```diff
@@ -18,6 +18,8 @@ class BootloaderFactory
       "grub2-efi"
     ].freeze
 
+    DEFAULT_KEYWORD = "default".freeze
```

Comment:
IMHO A comment would nice here, form the name it's hard to guess what that means...